pages/about/index.tsx

```diff
@@ -32,6 +33,54 @@ interface TeamSection {
   members: string;
 }
 
+type ArticleType = {
+  title: string;
+  url: string;
+  sourceName: string;
+  lang: string | null; // TODO Language type
```

Comment:
Actually I think `string` is fine here. `Language` type is reserved for languages that we support. :)